ui/preference/list_preference.go

```diff
@@ -27,10 +28,12 @@ type ListPreference struct {
 	clickable         *decredmaterial.Clickable
 	optionsRadioGroup *widget.Enum
 
+	cancelButton decredmaterial.IconButton
+
 	updateButtonClicked func()
 }
 
-func NewListPreference(wallet *wallet.Wallet, theme *decredmaterial.Theme, preferenceKey, defaultValue string, items map[string]string) *ListPreference {
+func NewListPreference(wallet *wallet.Wallet, theme *decredmaterial.Theme, preferenceKey, defaultValue string, items map[string]string, cancelIcon *widget.Icon) *ListPreference {
```

Comment:
```suggestion
func NewListPreference(wallet *wallet.Wallet, l *load.Load, preferenceKey, defaultValue string, items map[string]string) *ListPreference {
```

you will need to import the load package.